_includes/about.html

```diff
@@ -30,4 +30,11 @@ <h3>Imperative and reactive code</h3>
       <p>This is helpful for both Java developers who are used to working with the imperative model and don’t want to switch things up, and those working with a cloud-native/reactive approach. The Quarkus development model can adapt itself to whatever app you’re developing. <br><a href="/continuum/">Learn more about Reactive</a></p>
     </div>
   </div>
+  <h2>The inspiration behind Quarkus...</h2>
+    <p>Quarkus 1.0 was released back in November 2019. For more about the why, how and what of Quarkus, check out this presentation by Emmanuel Bernard given at Devoxx Belgium in 2019. Emmanuel works at Red Hat as a senior distinguished engineer and he is the co-lead/co-founder of the Quarkus project. He is also a Java Champion. Curious to know where the name "Quarkus" comes from? Check out Emmanuel's presentation!</p>
```

Comment:
```suggestion
    <p>For more about the why, how and what of Quarkus, check out the Devoxx Belgium in 2019 presentation where Quarkus 1.0 was released in November 2019.</p>
```
just to simplify the text - not sure about the placement hence why I bumped it out of draft.